src/azul/service/manifest_service.py

```diff
@@ -350,13 +369,13 @@ def source_filter(self) -> SourceFilters:
 
     @classmethod
     def for_format(cls,
-                   format_: str,
+                   format_: ManifestFormat,
                    service: ManifestService,
                    filters: Filters) -> 'ManifestGenerator':
         """
         Return a generator instance for the given format and filters.
 
-        :param format_: a string describing the format
+        :param format_: a member of ManifestFormat
```

Comment:
```suggestion
        :param format_: a member of ManifestFormat
```

Both fragments, the original and the changed version don't tell me anything I don't already know from looking at the signature.